cmd/kpod/parse.go

```diff
@@ -5,79 +5,22 @@ package main
 
 import (
```

Comment:
Leave the things in this file - we're slowly adding functionality to kpod run which requires them.

Reply:
done!